ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -21,21 +21,21 @@
 </div>
 
 <div class="mt-8 prose prose-lg text-gray-800 pb-6">
-  <h3 class="ds-h3">Examples of Correct use</h3>
+  <h3 class="ds-h3">Examples of correct use</h3>
   <CodeBlock @language="markup" @code="&lt;button aria-label=&quot;Check Activity&quot;>
  &lt;FlightIcon @name=&quot;activity&quot; />
 &lt;/button>" />
   <CodeBlock @language="markup" @code="&lt;h2>Activity Report &lt;FlightIcon @name=&quot;activity&quot; />&lt;/h2>" />
 </div>
 
 <div class="mt-8 prose prose-lg text-gray-800 pb-6">
-  <h2 class="ds-h2" id="ember-flight-icons"><a href="#ember-flight-icons" class="ds-a" rel="external">&sect;</a> Use in Ember Apps</h2>
+  <h2 class="ds-h2" id="ember-flight-icons"><a href="#ember-flight-icons" class="ds-a" rel="external">&sect;</a> Use in Ember apps</h2>
```

Comment:
We should figure out a writing consistency style, here's what I usually try to do: 

h1 & h2: capital case
h3 + : sentence case

WDYT?

Reply:
I'm used to seeing copy / typography style guides use the same casing throughout. Is that your personal preference or a standard from somewhere?

Reply:
We should try to align to HashiCorp's Voice, Style, and Language Guidelines. Will share link in Slack.